Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -101,6 +101,8 @@ void TrackerDPL::init(InitContext& ic)
     trackParams[0].ZBins = 64;
     trackParams[0].PhiBins = 32;
     trackParams[0].MinTrackLength = 4;
+    trackParams[0].CellsPerClusterLimit = 1.e2f;
+    trackParams[0].TrackletsPerClusterLimit = 1.e2f;
     LOG(info) << "Initializing tracker in sync. phase reconstruction with " << trackParams.size() << " passes";
```

Comment:
@mconcas @mpuccio After suppressing ITS reco prescaling in the sync. mode of FST, I got a message
```
Too many tracklets per cluster (3.18), check the detector status and/or the selections. Current limit is 2.0
```
To avoid this, I am setting the thresholds to higher values, still 10 times less than in the async mode. Please confirm that this is ok.

Reply:
> @mconcas @mpuccio After suppressing ITS reco prescaling in the sync. mode of FST, I got a message
> 
> ```
> Too many tracklets per cluster (3.18), check the detector status and/or the selections. Current limit is 2.0
> ```
> 
> To avoid this, I am setting the thresholds to higher values, still 10 times less than in the async mode. Please confirm that this is ok.

Changing that value would change the production setting we have in place for sync mode.
We are probably going to be safe anyway because of the prescaler, but we could think of letting the default as it is and overriding the configuration in FST with the higher value.